### PR TITLE
Fixes #35118 - add cvv compare button and basic page with Packages

### PR DESCRIPTION
**Pull request by tazhibaevaaliya**
#### What are the changes introduced in this pull request?
I added the "Compare" button to the ContentViewVersions page. This button directs to the table with selected versions and the  comparison (checks and crosses), showing the common and different packages in two versions
#### What are the testing steps for this pull request?
1. Select two versions on the ContentViewVersions page
2. Hit "Compare" button
3. The new table needs to be rendered with crosses and checks for each different and common packages respectively
4. Try changing the version in the dropdown and check if the table re-renders

### CODE REVIEW

File: app/controllers/katello/concerns/api/v2/repository_content_controller.rb [hunk 1]
```diff
@@ -48,6 +48,7 @@ def show
       param :content_view_version_ids, Array, :desc => N_("content view versions to compare")
       param :repository_id, :number, :desc => N_("Library repository id to restrict comparisons to")
       def compare
+        # params[:content_view_version_ids] = [41,42]
         fail _("No content_view_version_ids provided") if params[:content_view_version_ids].empty?
```
Comment (sjha4) [inline app/controllers/katello/concerns/api/v2/repository_content_controller.rb:52]: Delete this.. :D

File: webpack/scenes/ContentViews/ContentViewsConstants.js [hunk 2]
```diff
@@ -51,7 +52,7 @@ export const cvVersionPublishKey = (cvId, versionCount) => `${PUBLISH_CONTENT_VI
 export const cvVersionTaskPollingKey = cvId => `CONTENT_VIEW_VERSION_POLLING_${cvId}`;
 export const cvAddComponentKey = cvId => `${CONTENT_VIEWS_KEY}_ADD_COMPONENT_${cvId}`;
 export const cvRemoveComponentKey = cvId => `${CONTENT_VIEWS_KEY}_REMOVE_COMPONENT_${cvId}`;
-
+export const cvPackagesCompare = (versionOne, versionTwo) => `${PACKAGES_KEY}_${versionOne}_${versionTwo}`;
```
Comment (sjha4) [inline webpack/scenes/ContentViews/ContentViewsConstants.js:55]: Were you gonna change these names `cvPackagesCompare` to `cvPackagesComparison`

Comment (tazhibaevaaliya) [reply webpack/scenes/ContentViews/ContentViewsConstants.js:55]: @sjha4  Yeah, I was. I think I asked you about this last time. If I am not wrong, you said that I can keep all new names on the next PR. Is that so?

**Comment by tazhibaevaaliya** [comment]
Closing this PR in favor of a [new PR](https://github.com/Katello/katello/pull/10195), which contains all the features introduced in this PR.

**Pull request closed without merging**